### ISSUE TITLE
GLTFLoader: .aoMap broken when mesh has one interleaved UV set

### ISSUE THREAD

**Issue by donmccurdy** [mrdoob/three.js#19153]
three.js requires a 2nd UV set for AO textures, while glTF does not. To work around the common case where a model has only one set of UVs but the user expects their AO map to appear, GLTFLoader automatically clones the UVs when needed:

https://github.com/mrdoob/three.js/blob/eb8639016466f0066714f7c1c0c67a59898979e0/examples/js/loaders/GLTFLoader.js#L2071-L2077

That method of copying the UV set does not work if the attribute is interleaved; the resulting UVs will be corrupt. Related: https://github.com/mrdoob/three.js/issues/15395

**Comment by donmccurdy** [comment]
Example:

![Screen Shot 2020-04-16 at 6 13 52 PM](https://user-images.githubusercontent.com/1848368/79521446-10ebae80-800e-11ea-9f47-eafe5bd4c4c6.png)


https://drive.google.com/open?id=19UJvpSrrJg5vSxVpKt2kF44g1I8SPQt9

**Comment by WestLangley** [comment]
I would expect this to work:
```js
geometry.setAttribute( 'uv2', geometry.attributes.uv );
```

**Comment by Mugen87** [comment]
Just for clarification: The current approach is not really a deep copy. The array is shared. 

I thought  `GLTFLoader` would do this instead:
```js
geometry.setAttribute( 'uv2',  geometry.attributes.uv.clone() ); 
```
Of course won't work for  `InterleavedBufferAttribute`. However, I've pointed out earlier that `InterleavedBufferAttribute` should have a `clone()` and `toJSON()` method and just de-interleave data. If instead an instance of `BufferGeometry` with interleaved attributes is cloned, one could retain the interleaved data and return the same configuration of attributes.

**Comment by WestLangley** [comment]
>However, I've pointed out earlier that InterleavedBufferAttribute should have a clone()

I think we should be removing `clone()` from every method for which we are unable to implement it correctly. It is a mess, largely unneeded, and well-defined probably only for the math classes. I have made this point several times before -- but or course, this is not the thread to be discussing that.

I think the solution I suggested above is appropriate for this use case, and it worked in my limited testing.
